mongo/integration/mtest/mongotest.go

```diff
@@ -86,6 +86,9 @@ type WriteConcernErrorData struct {
 type T struct {
 	*testing.T
 
+	_               *bool // 64-bit align connsCheckedOut on 64-bit and 32-bit platforms.
+	connsCheckedOut int64 // net number of connections checked out during test execution
```

Comment:
Why the need for the `_` here? Can't we just move `connsCheckedOut` to the top of `T`?

Reply:
Yep, that's a better idea 👍